neoexchange/core/management/commands/upload_data.py

```diff
@@ -23,6 +23,7 @@
 from django.core.files.storage import default_storage
 
 from core.utils import search
+from neox.settings import USE_S3
```

Comment:
Think this should be `from django.conf import settings` to make it (a little) more general and also matches the other uses in e.g. `lightcurve_extraction`